apps/LineRider/install

```diff
@@ -1,19 +1,32 @@
 #!/bin/bash
 
-version=2022.5.3
 #Bot automatically updates this when I make a new release.
+version=2022.5.3
+#Buster version
+version2=2022.5.1
 
+status "Installing Depends... "
 install_packages apt-transport-https dirmngr gnupg ffmpeg mono-devel unzip zip || exit 1
+status_green "Done."
```

Comment:
These "Installing Depends... " and "Done." messages are unnecessary for the `install_packages` command, as it will already say what is being done. When `install_packages` finishes, it says "Package installation complete."

See the api line 518.

Reply:
Resolved: [e31ca13](https://github.com/Botspot/pi-apps/pull/2052/commits/e31ca13b030484b318ec881494da2eb9d2a52ae9)